docs/user-guide/configuring.md

```diff
@@ -81,7 +81,7 @@ The following parsers are compatible with ESLint:
 
 * [Esprima](https://www.npmjs.com/package/esprima)
 * [Babel-ESLint](https://www.npmjs.com/package/babel-eslint) - A wrapper around the [Babel](https://babeljs.io) parser that makes it compatible with ESLint.
-* [typescript-eslint-parser(Experimental)](https://www.npmjs.com/package/typescript-eslint-parser) - A parser that converts TypeScript into an ESTree-compatible form so it can be used in ESLint. The goal is to allow TypeScript files to be parsed by ESLint (though not necessarily pass all ESLint rules).
+* [@typescript-eslint/parser](https://www.npmjs.com/package/@typescript-eslint/parser) - An ESLint custom parser which leverages TypeScript ESTree to allow for ESLint to lint TypeScript source code. The goal is to allow TypeScript files to be parsed by ESLint (though not necessarily pass all ESLint rules).
```

Comment:
> The goal is to allow TypeScript files to be parsed by ESLint (though not necessarily pass all ESLint rules).

not sure it is correct at the moment?

Reply:
I personally think the current first sentence is a little more user-friendly, since the underlying implementation detail of using typescript-estree is probably not relevant to most users.

As for the second sentence, I'm not sure of that either! But I do think it's probably unnecessary for us to state that in our docs. Since the project has moved to another org, that mission statement might change over time.

Reply:
I, a non-English speaker, prefer the current sentence, though the changed was from that repo. 😄